sql/core/src/test/resources/sql-tests/results/postgreSQL/window_part1.sql.out

```diff
@@ -422,7 +421,7 @@ struct<ten:int,two:int,gsum:bigint,wsum:bigint>
 -- !query
 SELECT count(*) OVER (PARTITION BY four) FROM (SELECT * FROM tenk1 WHERE FALSE)s
```

Comment:
Okay, now I completely got what you're trying to do it. You do want _window functions_ to work without specifying the ordering, and non-window functions already work without specifying ordering (because the results will be deterministic anyway). Yes, -1 for the same comment from @hvanhovell.